src/main/MainMenu.ts

```diff
@@ -98,42 +99,52 @@ export const getDefaultMenuTemplate = (
     {
       label: 'View',
       submenu: [
+        {
+          label: 'Show Internal Realm features',
+          visible: showInternalFeatures || enableTogglingInternalFeatures,
```

Comment:
Any particular reason for wanting to disable the internal features? Testing? Since you special knowledge to enable them in the first place?

Reply:
Right - there's no particularly compelling reason to want to disable internal features - I guess if you're not really using them anymore, you may want to reduce clutter in the menus. And just felt awkward to have a feature you can enable, but not disable 😅

Reply:
No big issue, I'm fine with keeping it

Reply:
It's nice to be able to disable the internal features when you want to do a screenshot or screencast showing off a feature to regular users.